WebApplication.Tests/MigrationBucketKeyProvider.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading.Tasks;
+
+namespace WebApplication.Services
+{
+    public class MigrationBucketKeyProvider : IBucketKeyProvider
+    {
+        public string BucketKey { private get; set; }
+
+        public Task<string> GetBucketKeyAsync()
+        {
+            return new Task<string>(() => BucketKey);
```

Comment:
Would be better to use `Task.FromResult(BucketKey)`

Reply:
This file was on a wrong location, I have deleted it.